web/src/routes/search/containers/SearchPage.js

```diff
@@ -69,7 +73,10 @@ export class SearchPage extends React.Component<PropsType, LocalStateType> {
       .concat(categoriesWithContent)
   }
 
-  handleFilterTextChanged = (filterText: string) => this.setState({ filterText: filterText })
+  handleFilterTextChanged = (filterText: string) => {
+    this.setState({ filterText: filterText })
+    history.replaceState(null, '', `${this.props.location.pathname}?query=${filterText}`)
```

Comment:
Not sure whether something like `query-string.stringify({query: filterText}` would be better.

Additional observation: If filterText is empty, the url becomes `integreat.app/augsburg/de/search?query=`. Not sure if this should be handled separately...

Reply:
I think we  should remove the whole query string if query is empty.